VM_SETUP.md

```diff
@@ -0,0 +1,179 @@
+# MAAS Anvil Development VM Setup
+
+For the purpose of developing and testing MAAS Anvil, you will need (up to) three VM's. While one is enough to deploy a non-HA setup, features should be tested in HA. Here, we use LXD VMs.
+
+## LXD Network Setup
+
+First, create an LXD network as shown below. Each VM you create will be assigned an IP address in this network.
+
+```bash
+CONTROL_IP_RANGE="10.30.0.1"
```

Comment:
This is only a single IP, but range implies it should have the /24 appended.

Should this be something like:
`CONTROL_NETWORK_PREFIX="10.30.0"` ?

and then changing the key:value below to
`  ipv4.address: ${CONTROL_NETWORK_PREFIX}.1/24`